plugins/modules/aci_syslog_group.py

```diff
@@ -18,47 +20,73 @@
 options:
   admin_state:
     description:
-    - Administrative state of the syslog group
+    - Administrative state of the syslog group.
     type: str
     choices: [ enabled, disabled ]
   console_logging:
     description:
-    - Log events to console
+    - Log events to console.
     type: str
     choices: [ enabled, disabled ]
   console_log_severity:
     description:
-    - Severity of events to log to console
+    - Severity of events to log to console.
+    - If unset during creation, value defaults to C(alerts).
     type: str
-    choices: [ alerts, critical, debugging, emergencies, error, information, notifications, warnings ]
+    choices: [ alerts, critical, emergencies]
   local_file_logging:
     description:
-    - Log to local file
+    - Log to local file.
     type: str
     choices: [ enabled, disabled ]
   local_file_log_severity:
     description:
-    - Severity of events to log to local file
+    - Severity of events to log to local file.
+    - If unset during creation, value defaults to C(alerts).
     type: str
     choices: [ alerts, critical, debugging, emergencies, error, information, notifications, warnings ]
   format:
     description:
-    - Format of the syslog messages. If omitted when creating a group, ACI defaults to using aci format.
+    - Format of the syslog messages.
+    - If unset during creation the value defaults to C(aci).
+    - C(rfc5424-ts) is only available starting from ACI version 5.2(8).
```

Comment:
```suggestion
    - C(rfc5424_ts) is only available starting from ACI version 5.2(8).
```
Also fix any other references to it